lib/transforms/gzip.js

```diff
@@ -9,25 +10,23 @@ module.exports = queryObj => {
 
     if (gzipAssets.length > 0) {
       try {
-        compress = require('node-zopfli');
+        compress = promisify(require('node-zopfli'));
       } catch (e) {
         assetGraph.info(
           new Error(
             'node-zopfli is not available, using less efficient zlib compression'
           )
         );
-        compress = require('zlib');
+        compress = promisify(require('zlib'));
       }
     }
 
-    await Promise.map(
+    await Bluebird.map(
       gzipAssets,
       async asset => {
         // http://webmasters.stackexchange.com/questions/31750/what-is-recommended-minimum-object-size-for-gzip-performance-benefits
         if (asset.type !== 'SourceMap' && asset.rawSrc.length > 860) {
-          const gzippedRawSrc = await Promise.fromNode(cb =>
-            compress.gzip(asset.rawSrc, cb)
-          );
+          const gzippedRawSrc = await compress.gzip(asset.rawSrc);
```

Comment:
Hmm, does this work? It seems weird that `compress.gzip` would return a promise when only the main export of `node-zopfli` or `gzip` has been promisified?

We don't seem to have any tests covering this transform.

Reply:
Good catch. Fixed in aa133bbb

Reply:
Great! I added a gzip test on master (https://github.com/assetgraph/assetgraph/commit/eb007153b08ec91149bd684ff0b0a2dbb307f559), and it also passes on this branch :muscle: